Source/Core/InputCommon/InputConfig.cpp

```diff
@@ -117,3 +117,17 @@ bool InputConfig::ControllersNeedToBeCreated() const
 {
 	return m_controllers.empty();
 }
+
+bool InputConfig::IsControllerAKeyboard(int index) const
```

Comment:
This name is pretty misleading considering the comment below is

`// Find anything which obviously not a controller` (which also contains a typo).